docs/skopeo-sync.1.md

```diff
@@ -68,6 +68,15 @@ Print usage statement.
 
 **--append-suffix** _tag-suffix_ String to append to destination tags.
 
+**--digestfile** _path_
+
+After copying the images from source, write the digest of the resulting images along with Image Reference.
+
+```
+bf91f90823248017a4f920fb541727fa8368dc6cf377a7debbd271cf6a31c8a7 <dest-repository>/alpine:edge
+31603596830fc7e56753139f9c2c6bd3759e48a850659506ebfb885d1cf3aef5 <dest-repository>/postgres:14.3
```

Comment:
`<dest-repository>` is not really accurate — `docker://…` for registry locations, and there is no “repository” for directory destinations.